Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:latest
+MAINTAINER Harshit Prasad
+
+# Update
+RUN apt-get update
+RUN apt-get upgrade -y
+
+# add packages
+RUN apt-get install -y git openjdk-8-jdk
+
+#install gradle required for build
+RUN add-apt-repository ppa:cwchien/gradle
+RUN apt-get update
+RUN apt-get install gradle
+
+# install apache ftp server 1.1.0
+RUN wget http://www-eu.apache.org/dist/mina/ftpserver/1.1.0/dist/apache-ftpserver-1.1.0.tar.gz
+RUN tar xfz apache-ftpserver-1.1.0.tar.gz
+
+# run ftp server
+RUN cd apache-ftpserver-1.1.0 bin/ftpd.sh res/conf/ftpd-typical.xml
+
+# install RabbitMQ server
+RUN wget https://www.rabbitmq.com/releases/rabbitmq-server/v3.6.6/rabbitmq-server-generic-unix-3.6.6.tar.xz
+RUN tar xf rabbitmq-server-generic-unix-3.6.6.tar.xz
+
+# run the RabbitMQ server
+RUN cd rabbitmq_server-3.6.6/sbin/rabbitmq-server
+
+# install erlang language for RabbitMQ
+RUN apt-get install erlang
+
+# install the management plugin to be able to use a web interface
+RUN rabbitmq_server-3.6.10/sbin/rabbitmq-plugins enable rabbitmq_management
```

Comment:
add these lines
`[{rabbit, [{loopback_users,[]}]}].` to rabbitmq_server-3.6.6/etc/rabbitmq/rabbitmq.config using `echo` command .